src/components/Select.vue

```diff
@@ -969,15 +969,31 @@
        * @return {void}
        */
       onSearchBlur() {
+
         if (this.mousedown && !this.searching) {
           this.mousedown = false
         } else {
           if (this.clearSearchOnBlur) {
             this.search = ''
           }
-          this.open = false
-          this.$emit('search:blur')
+          this.closeSearchOptions()
+        }
+
+        // Fixed bug where no-options message could not be closed
+        if(this.search.length === 0 && this.options.length === 0){
```

Comment:
so when a search query doesn't exist, and there are no options available, close the search? Is this maybe an `else if` to be joined up with the above conditional, I'm thinking otherwise `closeSearchOptions` could be called twice.

Reply:
Yeah good point. Maybe return void after the first time closeSearchOptions has been called? Seems more readable for me then.